ci-operator/config/openshift/ansible-operator-plugins/openshift-ansible-operator-plugins-main.yaml

```diff
@@ -19,6 +19,10 @@ images:
       as:
       - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.22-openshift-4.17
   to: openshift-ansible-operator-plugins
+- dockerfile_path: openshift/ci/dockerfiles/ansible-e2e.Dockerfile
+  # `from` replaces the the last FROM in the Dockerfile
+  from: openshift-ansible-operator-plugins
+  to: ansible-operator-e2e
 promotion:
```

Comment:
wondering if we need to exclude `ansible-operator-e2e` image, similar to 
https://github.com/openshift/release/blob/eaf6f91e499419014925405f49e43e4de6296890/ci-operator/config/openshift/ocp-release-operator-sdk/openshift-ocp-release-operator-sdk-master.yaml#L65-L71